README.md

```diff
@@ -1,8 +1,307 @@
+>>> Default wallet: t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq
+```
+This information will be relevant to configure our agent to connect to this rootnet node. For this, the `config.toml` section should be updated accordingly. In the above case, we need to set the endpoint of our rootnet node to be `127.0.0.1:1235`, we need to set `auth_token` to the one provided by the script, and the default account, for instance, the one provided by the script (although we could use ant other).
+
+The configuration for our rootnet should look therefore like this:
+```toml
+[subnets."/root"]
+id = "/root"
+jsonrpc_api_http = "http://127.0.0.1:1235/rpc/v1"
+jsonrpc_api_ws = "wss://example.org/rpc/v0"
+auth_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.j94YYOr8_AWhGGHQd0q8JuQVuNhJA017SK9EUkqDOO0"
+accounts = ["t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq"]
+```
+
+> Beware: If you are already running the daemon, changes in the config file are only picked up after running `./bin/ipc_agent reload-config` so be sure to run it after editing your config.
+
+Finally, to test if the connection to the rootnet has been successful, we can for instance try to create a new wallet in the rootnet: 
+```
+./bin/ipc_agent wallet-new --key-type=bls --subnet=/root
+```
+
+### Running a subnet
+To spawn a new subnet, our IPC agent should be connected to at least the subnet of the parent we want to spawn the subnet from. You can refer to the previous section for information on how to run or connect to a rootnet. This instructions will assume the deployment of a subnet from `/root`, but the steps are equivalent for any other parent subnet. 
+
+#### Spawn subnet actor
+To run a subnet the first thing is to configure and create the subnet actor that will govern the subnet's operation:
+```bash
+./bin/ipc_agent create-subnet -p <parent-id> -n <subnet-name> --min-validator-stake 1 --min-validators <num-validators> --finality-threshold <number-epochs> --check-period <epochs-between-checks>
+
+# Sample command execution
+./bin/ipc_agent create-subnet -p /root -n test --min-validator-stake 1 \
+--min-validators 0 --finality-threshold 10 --check-period 10
+
+[2023-03-21T09:32:58Z INFO  ipc_agent::cli::commands::manager::create] created subnet actor with id: /root/t01002
+```
+This command deploys a subnet actor for a new subnet from the `root`, with a human-readable name `test`, that requires at least `1` validator to join the subnet to be able to mine new blocks, and with a checkpointing period to the parent of `10` blocks. We can see that the output of this command is the ID of the new subnet.
+
+#### Exporting wallet keys from subnet
+In order to run a validator in a subnet, we'll need a set of keys to handle that validator. To export the validator key from a wallet that may live in some other subnet into a file (like the wallet address we are using in the rootnet), we can use the following Lotus command:
```

Comment:
```suggestion
#### Exporting wallet keys
In order to run a validator in a subnet, we'll need a set of keys to handle that validator. To export the validator key from a wallet that may live in another network (like the wallet address we are using in the rootnet), we can use the following Lotus command:
```

This is not objectively wrong and we may want to keep as is, but:

In the design doc, we define the rootnet as a subnet without parent. Under that definition, this is clear. But I wonder if we're leading the user astray in the context of the guide by suggesting this section is about exporting _subnet_ keys.

Reply:
Good point. Fixed. (btw, I am editing instead of committing suggestions so I can change other things in the meantime, will push all at once when it is ready, in case you don't see the change reflected immediately).